src/python/sortingview/workspace/workspace.py

```diff
@@ -322,10 +322,13 @@ def _get_sorting_curation(subfeed: kc.Subfeed, sorting_id: str):
     subfeed.set_position(0)
     labels_by_unit = {}
     merge_groups = []
+    is_closed = False
```

Comment:
I assume you are going to implement the close/reopen functionality in a different PR?